replica/database.cc

```diff
@@ -1034,6 +1034,8 @@ bool database::update_column_family(schema_ptr new_schema) {
 void database::remove(const table& cf) noexcept {
     auto s = cf.schema();
     auto& ks = find_keyspace(s->ks_name());
+    table& mutable_cf = const_cast<table&>(cf);
```

Comment:
Perhaps I should change the signature of `database::remove` to not take a const param, but a mutable one - after all, the table object is changed here. On the other hand, this is a small and neglectable change.